### DISCUSSION TITLE
String concatenation operator

### DISCUSSION THREAD

**Discussion by FranKusmiruk** [SkriptLang/Skript#3975] [Suggestions]
**Proposed by**: Goose (Goose#1832)

### Description
Ability to use + as a concatenation operator for strings

### Example Usage
```lua
send "hey" + " there!" to player # "hey there!"
send "&7You have &e" + {coins::%player's uuid%} + " &7coins!" # "&7You have &e%{coins::%player's uuid%}% &7coins!"
```

**Comment by FranKusmiruk** [comment]
My take on this is that it is potentially useless considering Skript makes use of string templating to achieve this behavior but if it does get added I'd rather see it with a different operator.

BaeFell proposed a dot like PHP does but we'd also have to take into account whether that'd conflict with addons such as skript-reflect, considering how massively it is used we may prefer another symbol for the operator.

**Reply by ghost** [reply]
In that case, how about double dots like in lua? 
`"string".."string"`

**Comment by TPGamesNL** [comment]
I don't think this should be added. It could've been useful if we weren't able to fix quote parsing (#590), but now that that's been fixed, I don't think this'd provide much benefit.

**Comment by Mwexim** [comment]
I'd suggest using `send join "Hello " and "there" to player` for anyone wondering.

While this syntax is not verbose at all and *should* probably be enhanced, it is a valid work-around, since the delimiter is optional anyway.
